internal/controllers/common/resource_definitions/resource_definitions.go

```diff
@@ -774,6 +900,35 @@ func NewGrafanaService(cr *operatorv1beta1.Cryostat) *corev1.Service {
 	}
 }
 
+func NewReportService(cr *operatorv1beta1.Cryostat) *corev1.Service {
+	serviceType := corev1.ServiceTypeLoadBalancer
+	config := &operatorv1beta1.ServiceConfig{
+		ServiceType: &serviceType,
+	}
+	configureService(config, cr.Name, "reports")
+	return &corev1.Service{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:        cr.Name + "-reports",
+			Namespace:   cr.Namespace,
+			Labels:      config.Labels,
+			Annotations: config.Annotations,
+		},
+		Spec: corev1.ServiceSpec{
+			Type: *config.ServiceType,
+			Selector: map[string]string{
+				"app":       cr.Name,
+				"component": "reports",
+			},
+			Ports: []corev1.ServicePort{
+				{
+					Port:       reportsPort,
+					TargetPort: intstr.IntOrString{IntVal: reportsPort},
+				},
+			},
+		},
+	}
+}
+
```

Comment:
This should be customizable like the other services are now: https://github.com/cryostatio/cryostat-operator/blob/0ee99b4242c5b53d2186ce56ec5af6531179b36f/api/v1beta1/cryostat_types.go#L111-L144

We can do this in a followup PR if you prefer.

Reply:
If this and the cert-manager integration are the only two major changes to be done then I don't mind doing this change as part of this PR, and cert-manager in a follow-up.